modules/keepalived-limitation-for-ipi.adoc

```diff
@@ -0,0 +1,73 @@
+[discrete]
+[id="keepalived-limitation-for-ipi_{context}"]
+= Identifying virtual router conflicts
+
+The Openshift cluster Virtual IPs are managed using multicast (VRRPv2 or VRRPv3), and there is a limitation of 255 unique virtual routers per multicast domain. Each VIP has a virtual router-id associated with it based on the cluster name. Duplication of these ids can result in a conflict, preventing the Virtual IPs from being assigned to the node. Therefore, if you have multiple clusters using the same network, you should check to see which VIPs the installation will choose to avoid conflicts.
```

Comment:
Maybe we should clarify that this step is required only if you have multiple clusters using the same network.
So maybe something like:

"""
This step is only required if you have multiple clusters using the same network.

The Openshift cluster Virtual IPs are managed using multicast (VRRPv2 or VRRPv3), and there is a limitation of 255 unique virtual routers per multicast domain. Each VIP has a virtual router-id associated with it based on the cluster name. Duplication of these ids can result in a conflict, preventing the Virtual IPs from being assigned to the node. Therefore, if you have multiple clusters using the same network, you should check to see which VIPs the installation will choose to avoid conflicts.
"""

Reply:
Done.